RELEASING.md

```diff
@@ -0,0 +1,99 @@
+# Releasing Sourcery
+
+There're no hard rules about when to release Sourcery. Release bug fixes frequently, features not so frequently and breaking API changes rarely.
+
+### Release
+
+Example is for releasing `0.6.1` version of the Sourcery. Make sure you've been added as owner for CocoaPods and have set up the api token to be able to upload releases on GitHub.
+
+Install dependencies.
+```
+bundle install
+bundle exec pod install
+```
+
+Run tests, check that all tests succeed locally and prepare the build.
+```
+rake tests
```

Comment:
I think just running `rake release:new` would be enough, it runs the tests and checks docs as a subtask (we can add concrete instructions in check docs error message)